multiapps-controller-shutdown-client/src/main/java/org/cloudfoundry/multiapps/controller/shutdown/client/ShutdownClientImpl.java

```diff
@@ -18,7 +18,7 @@
 class ShutdownClientImpl implements ShutdownClient {
 
     private static final String X_CF_APP_INSTANCE = "x-cf-app-instance";
-    private static final String SHUTDOWN_ENDPOINT = "/admin/shutdown";
+    private static final String SHUTDOWN_ENDPOINT = "/rest/admin/shutdown";
```

Comment:
This change is actually because ShutdownClientImpl is outdated and this is the new endpoint. In fact it is not used in the lss.

Reply:
As I remember, in LSS we make some calls to shutdown flowable job executors instance by instance.

Reply:
yes, of course, but this endpoint is in the shutdown CLIENT, which we do not use productively. This is the one in `multiapps-controller-shutdown-client` module, which I used to test our graceful shutdown and I noticed it is outdated (apparently because it is not used)

Reply:
okay, agreed